### DISCUSSION TITLE
Credential profiles for an easier world

### DISCUSSION THREAD

**Discussion by vongohren** [decentralized-identity/interoperability#63] [Topics]
@TelegramSam Lets test out this and see where we can collect the nessecary resources and plan a time for next meet

Here we can vote up the most important conversations and topics so that we keep this floated

Notes from IIW: https://docs.google.com/document/d/1ex_6aUqUOKEkOm2czOIlgSZZGwXKpdhy6G4_ARsHmrc/edit

**Comment by vongohren** [comment]
I wanted to document this issue as a clear problem on changing context. I was totally oblivious to this and this happened because I cached the context at two different points, and there have been no versioning nor hashlinking involved:

https://github.com/vongohren/verify-issues/pull/2

There one can also see the clear context changes that made things work!

**Comment by TelegramSam** [comment]
Just a note to indicate this has not been forgotten.

**Reply by vongohren** [reply]
Great to hear @TelegramSam
